content/en/news/releases/1.6.x/announcing-1.6/announcing-1.6.1/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.6.1
+linktitle: 1.6.1
+subtitle: Patch Release
+description: Istio 1.6.1 patch release.
+publishdate: 2020-06-03
+release: 1.6.1
+aliases:
+    - /news/announcing-1.6.1
+---
+
+This release contains bug fixes to improve robustness .... This release note describes
+what’s different between Istio 1.6.0 and Istio 1.6.1.
+
+{{< relnote >}}
+
+## Changes
+- **Fixed** support for Pod annotations to override mesh-wide Proxy settings
+- **Fixed** support for all Envoy filters supported in [Envoy's go-control-plane](https://github.com/envoyproxy/go-control-plane) ([Issue 23909](https://github.com/istio/istio/issues/23909))
```

Comment:
The phrasing of this seems odd to me. It seems to say that every Envoy filter has been fixed.

Reply:
Perhaps 
```suggestion
- **Updated** `EnvoyFilter` to register all filter types in order to support `typed_config` attributes ([Issue 23909](https://github.com/istio/istio/issues/23909))
```

Reply:
I'm going to accept your suggestion, but it still needs some re-work.